### PR TITLE
WIP:Add riops forecast dataset

### PR DESCRIPTION
**Pull request by racetted**
First draft, still need to review some of the parameters. 

Contains a fix to the test.yml file for a bad url.

### CODE REVIEW

File: metadata/mcf/msc_nwp_riopsf.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+              crs: 4326
+            - bbox: [-2500.000, -2500.000, 8847500.000, 8047500.000]
+              proj4: "+proj=stere +lat_0=90 +lon_0=-100 +k=0.93301243 +x_0=4245000 +y_0=5295000 +R=6371229 +units=m +no_defs"
+        temporal:
+            - begin: 2016-06-21
+              end: null
+              resolution: P3H
+    url: https://met5node.weather.gc.ca/catalogue
+
+distribution:
```
Comment (tomkralidis) [inline metadata/mcf/msc_nwp_riopsf.yml:56]:
```yaml
type: application/geo+json
rel: items
```

File: metadata/mcf/msc_nwp_riopsf.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+
+base_mcf: /share/contact-main.yml
+
+mcf:
+    version: 1.0
+
+metadata:
+    charset: utf8
+    hierarchylevel: dataset
+    identifier: urn:wmo:md:${MET5_CENTRE_ID}:ocean:medium-range:nwp.riops
```
Comment (tomkralidis) [inline metadata/mcf/msc_nwp_riopsf.yml]: `s/MET5_CENTRE_ID/WIS2_CENTRE_ID/g`

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: metadata/mcf/msc_nwp_riopsf.yml (hunk 2 above)
Comment (racetted) [reply metadata/mcf/msc_nwp_riopsf.yml]: Intentional change, didn't think we'd reuse the exact same ones, but I can be mistaken.

Comment (tomkralidis) [reply metadata/mcf/msc_nwp_riopsf.yml]: We are using the identical centre id as WIS2 proper, yes.

Comment (racetted) [reply metadata/mcf/msc_nwp_riopsf.yml]: Ok, noticed your github action, adjusting.

**Pull request merged**